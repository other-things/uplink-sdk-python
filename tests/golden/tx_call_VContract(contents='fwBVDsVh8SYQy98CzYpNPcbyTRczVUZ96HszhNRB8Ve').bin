00000000: 00 00 00 01 09 F9 55 4A  E9 80 4C 33 00 AB CA 52  ......UJ..L3...R
00000010: 3A 63 6A 53 AF 81 F6 39  AD 6E BA 17 0E 05 18 F0  :cjS...9.n......
00000020: 3F E2 9E 29 00 00 00 00  00 00 00 03 67 65 74 00  ?..)........get.
00000030: 00 00 00 00 00 00 01 04  09 F9 55 4A E9 80 4C 33  ..........UJ..L3
00000040: 00 AB CA 52 3A 63 6A 53  AF 81 F6 39 AD 6E BA 17  ...R:cjS...9.n..
00000050: 0E 05 18 F0 3F E2 9E 29                           ....?..)