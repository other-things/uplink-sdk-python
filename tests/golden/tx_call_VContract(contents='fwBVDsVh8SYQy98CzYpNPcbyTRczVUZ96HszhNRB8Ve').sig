r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 37154903807445325931863213639416942972776323539776523345219628426945745823146
