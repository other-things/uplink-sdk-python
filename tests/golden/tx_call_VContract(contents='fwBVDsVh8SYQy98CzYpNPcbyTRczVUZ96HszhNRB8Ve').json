{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "args": [
                    {
                        "contents": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                        "tag": "VContract"
                    }
                ],
                "method": "get"
            },
            "tag": "Call"
        },
        "tag": "TxContract"
    },
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE0zNzE1NDkwMzgwNzQ0NTMyNTkzMTg2MzIxMzYzOTQxNjk0Mjk3Mjc3NjMyMzUzOTc3NjUyMzM0NTIxOTYyODQyNjk0NTc0NTgyMzE0Ng=="
}